### ISSUE TITLE
Early Development Phase 

### ISSUE THREAD

**Issue by vipulgupta2048** [pyconraj/pyconraj2019#2]
Before launching, we need to smoothen things out for contributors to come in and work easily. 

- [ ] Add README @vipulgupta2048
- [ ] Add License @thisisayusha
- [ ] Add Contributing @vipulgupta2048
- [ ] Create Code of Conduct for contributing  @Shivank98  
- [ ] Create Code of Conduct for the conference (Refer Pycon India 2018 - @Shivank98)
- [ ] Add Team members in the pyconraj organisation @Shivank98

**Comment by thisisayush** [comment]
@vipulgupta2048  What license do you propose? Pycon 2018 used a variant of CC License.

**Comment by vipulgupta2048** [comment]
Yes, PyCon India uses [this](https://github.com/pythonindia/inpycon2018/blob/master/LICENSE.md) as their license. I say we can keep it simple by keeping the MIT License or go with some strong copylefts. The decision goes back to the organization and the person who wrote the code. For now, let's focus on getting the site online and worry about licenses later.
